newrelic-python.ini

```diff
@@ -51,7 +51,7 @@ monitor_mode = true
 # To completely disable security set flag to false If the flag is
 # set to false, the security module is not loaded. This property
 # is read only once at application start.
-security.agent.enable = agent_enabled_flag
+security.agent.enabled = agent_enabled_flag
 
 # Indicates if attack detection security module is to be enabled
 security.enable = security_enabled_flag
```

Comment:
security.enable should be security.enabled

Reply:
Updated